lib/instrument/transformer/nskeyed.js

```diff
@@ -483,7 +483,7 @@ class Archive {
         return this.encodeDict(obj, archiveObj);
       }
     } else {
-      throw Error(`Unable to encode types: ${typeof obj}`);
+      throw new Error(`Unable to encode types: ${typeof obj}`);
```

Comment:
:+1: for the fix, but otherwise this is a pretty useless message.  maybe something like:

```suggestion
      throw new Error(`Unable to encode types: ${util.format('%O', obj)}`);
```

where `util` is `import util from 'node:util'`

Reply:
This actually should be a `obj.constructor.name`, I could fix them